README.md

```diff
@@ -22,9 +42,9 @@ If the issue is a protocol weakness that cannot be immediately exploited or some
 
 ## Install
 
-The canonical download instructions for IPFS are over at: http://ipfs.io/docs/install/
+The canonical download instructions for IPFS are over at: http://ipfs.io/docs/install/. It is **highly suggested** you follow those instructions if you are not interested in working on IPFS development. Otherwise, follow the instructions below.
```

Comment:
"below" could be more specific (which below?), e.g. "intructions for [#build-from-source](building from source)"